rero_mef/marctojson/do_gnd_concepts.py

```
@@ -211,6 +211,23 @@ def trans_gnd_relation(self):
             if value:
                 self.json_dict[relation] = value
 
+    def trans_gnd_classification(self):
+        """Transformation classification from field 686."""
+        if self.logger and self.verbose:
+            self.logger.info("Call Function", "trans_gnd_classification")
```

Ruff (0.15.6)

[error] Too many arguments for `logging` format string

(PLE1205)